### PR TITLE
Fix UI ('Quoted' label)

### PR DESCRIPTION
**Pull request by audreyleow**
Changed the backgroundColour (gentler shade of blue), font color (black for consistency), fontSize (reduced) , added a boxShadow and fontWeight (bolded). I did not apply the blur function as it did not seem to improve the readability or aesthetic of the label.

### CODE REVIEW

File: src/pages/Browse.jsx [hunk 1]
```diff
@@ -238,10 +238,12 @@ export function Browse(props) {
                             position: "absolute",
                             top: 30,
                             right: 0,
-                            backgroundColor: "#2196f3",
+                            backgroundColor: "#9accf3b5",
                             padding: "10px 20px 10px 20px",
-                            fontSize: 18,
-                            color: "#ffffff"
+                            fontSize: 16,
+                            color: "rgb(0, 0, 0)",
```
Comment (elenacyl) [inline src/pages/Browse.jsx:244]: Let's standardize and use hex color codes instead of RGB just to keep it consistent throughout :)

Comment (audreyleow) [reply src/pages/Browse.jsx]: I've changed it in my latest commit! :)

**Comment by elenacyl** [comment]
image

Are we able to make the labels look like this, but instead of a gradient pink background, let’s find a gradient blue background with white fonts (google css gradient). We can leave it as a rectangle shape in the same position, but maybe add a border radius on the left edges of the rectangle?

**Comment by audreyleow** [comment]
> image
> 
> Are we able to make the labels look like this (see pink gradient label on top right), but instead of a gradient pink background, let’s find a gradient blue background with white fonts (google css gradient). We can leave it as a rectangle shape in the same position, but maybe add a border radius on the left edges of the rectangle?

I've made the above amendments in the latest commit! :)

**Pull request merged**